src/common/data/stacking-ref.ts

```diff
@@ -0,0 +1,118 @@
+import { renderMdx } from '@common/data/mdx';
+import STACKING_REFERENCE from '../../_data/boot-contracts-reference.json';
+
+const wrapInClarityTicks = (string: string) => {
+  let newString = '```clarity';
+  newString += `
+`;
+  newString += string.trim();
+  newString += `
+`;
+  newString += '```';
+  return newString;
+};
+
+const inlineCode = (string: string) => {
+  let newString = '`';
+  newString += string.trim();
+  newString += '`';
+
+  if (newString === '``') {
+    newString = '';
+  }
+
+  return newString;
+};
+
+const generateMarkdown = () => {
+  let publicFunctions = '';
+  let readonlyFunctions = '';
+  let errorCodes = '';
+
+  STACKING_REFERENCE.pox.public_functions.forEach(entry => {
+    publicFunctions += `
+### ${entry.name}
+
+**Signature:** ${inlineCode(entry.signature)}
+
+
+**Input:** ${inlineCode(entry.input_type)}
+
+
+**Output:** ${inlineCode(entry.output_type)}
```

Comment:
we should try `wrapInClarityTicks` here so that we get formatting. I'd give it a try else where, too, so we can format it. Some of these outputs are very long and would benefit from the formatting.

Reply:
I just gave it a try but it doesn't seem to improve UX. The way the input/output/signatures are formatted doesn't line up with clarity formatting rules, so the only benefit is that brackets are in a different color than the keys. The scrolling on one line could also make it challenging. I'd suggest we punt that for this PR and take note that we should redesign the reference page